packages/wrangler/src/__tests__/configuration.test.ts

```diff
@@ -1739,12 +1739,10 @@ describe("normalizeAndValidateConfig()", () => {
 		describe("[experimental_assets]", () => {
 			it("should override `experimental_assets` config defaults with provided values", () => {
 				const expectedConfig: RawConfig = {
-					experimental_assets: [
-						{
-							directory: "public/",
-							binding: "ASSETS",
-						},
-					],
+					experimental_assets: {
```

Comment:
as per our sync call today, moving away from array support, as this is not something for this iteration